src/api/v1/api.json

```diff
@@ -885,6 +890,23 @@
             "totalVolunteerHours": "Number",
             "volunteerLogs": ["types.volunteerLog"]
           }
+        },
+        "POST": {
+          "description": "Create volunteer logs for own community business",
```

Comment:
How do you know which user this log is attached to?

Reply:
Can admins create logs for other volunteers? If so, we can add an optional `userId` field in the payload.

Reply:
erm, I've encountered an issue:

`VOLUNTEER` and `VOLUNTEER_ADMIN` users are both given the `volunteer_logs-parent:write` scope. `ORG_ADMIN` users are given the `volunteer_logs-own:write` scope.

We want `VOLUNTEER_ADMIN` and `ORG_ADMIN` users to be able to add logs for other volunteers at their organisation. But, as mentioned, `VOLUNTEER_ADMIN` and `VOLUNTEER` users have the same scope, so it's not possible to prevent `VOLUNTEER` users from being able to add logs for other users purely through scopes.

The kinda gross solution to this is to simply check the user role in the handler. The reason I think this is kinda gross is because we have been trying to avoid explicit reference to roles in handlers where possible. I'm happy to do it for now, but if you have scope-related idea to solve this, I'd be interested in hearing it.

Reply:
My initial response it to change the concept of `logs` to be owned by the user who they’re related to. Then we would have `volunteer_logs-sibling` for `VOLUNTEER_ADMIN` and `volunteer_logs-own` for `VOLUNTEER`s

But I’ll have a think if there’s another solution.

Reply:
I think this needs more thought, so perhaps worth continuing with the bad solution and addressing this separately. Can open an issue.

Reply:
@eliascodes can you comment all areas this fix is implemented with a `//TO BE REPLACED`

Then happy to merge this